samwise.js

```diff
@@ -1,58 +1,53 @@
-/**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.